girder/models/assetstore.py

```diff
@@ -120,7 +120,16 @@ def addComputedInfo(self, assetstore):
         from .file import File
         from girder.utility import assetstore_utilities
 
-        adapter = assetstore_utilities.getAssetstoreAdapter(assetstore)
+        try:
+            adapter = assetstore_utilities.getAssetstoreAdapter(assetstore)
+        except GirderException as e:
```

Comment:
I think the right thing to do here is to subclass `GirderException`, and only catch instances of that subclass. That won't break the existing behavior, since the subclass still is an instance of `GirderException`.

Reply:
No where else do we subclass `GirderException`, which surprised me.  Oddly, we have this concept of an identifier in that exception (which, clearly, the subclass could use).

Reply:
That sounds like a good idea. The subclass should be able define its identifier at the class-level, rather than in every constructor call.

IMO, we should drop the identifiers entirely, but we can always do that globally in a future pass, with more developer consensus.